pyrogram/client/types/messages_and_media/webpage.py

```diff
@@ -0,0 +1,192 @@
+# Pyrogram - Telegram MTProto API Client Library for Python
+# Copyright (C) 2017-2019 Dan Tès <https://github.com/delivrance>
+#
+# This file is part of Pyrogram.
+#
+# Pyrogram is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Pyrogram is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Pyrogram.  If not, see <http://www.gnu.org/licenses/>.
+
+import pyrogram
+from pyrogram.api import types
+from ..object import Object
+
+
+class WebPage(Object):
+    # TODO: hash, cached_page
+    """A webpage preview
+
+    Parameters:
+        id (``str``):
+            Unique identifier for this webpage.
+
+        url (``str``):
+            Full URL for this webpage.
+
+        display_url (``str``):
+            Display URL for this webpage.
+
+        type (``str``, *optional*):
+            Type of webpage preview, known types (at the time of writing) are:
+            `article`, `photo`, `gif`, `video` and `document`,
+            `telegram_user`, `telegram_bot`, `telegram_channel`, `telegram_megagroup`.
```

Comment:
While I strongly prefer having comprehensive docs, in this case we have no way of telling what all the possible values are (unless maybe we look at official clients/libs). This approach is good enough, except the style you used. Maybe it's better to use \*italic* instead, like I did in other places.

Reply:
f7bfd55

Reply:
Is this ok?